pkgs/top-level/all-packages.nix

```diff
@@ -11752,7 +11752,7 @@ with pkgs;
     boost = boost179;
     libclang = llvmPackages_15.libclang;
     clang =
-      if stdenv.cc.libcxx != null
+      if stdenv.cc.isClang
```

Comment:
`stdenv.cc.libcxx.pname` should be checked. Not all `clang` based `stdenv`s use libcxx which would be the case in the `else` condition then…